File: examples/twitter/user/node/getAUsersTwitterProfile.js
const { TwitterService } = require("micro-js-client/twitter");

// Get a user's twitter profile
async function getAusersTwitterProfile() {
  let twitterService = new TwitterService(process.env.MICRO_API_TOKEN);
  let rsp = await twitterService.user({
    username: "crufter",
  });
  console.log(rsp);
}

getAusersTwitterProfile();

File: examples/go/twitter/user/getAUsersTwitterProfile.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/twitter"
)

// Get a user's twitter profile
func GetAusersTwitterProfile() {
	twitterService := twitter.NewTwitterService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := twitterService.User(&twitter.UserRequest{
		Username: "crufter",
	})
	fmt.Println(rsp, err)
}
